### DISCUSSION TITLE
后AI时代 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#425] [General]
# 后AI时代 - Miao Yu | 于淼

https://yufree.cn/cn/2023/07/19/post-ai/

**Comment by cxy0714** [comment]
眼神中只有慈爱没有仇恨，哈哈哈，好生动。
另外，围棋中的定式是否也可以看做只是经验总结的**平均**的局部最优的方案（围棋盲，只看过b站up主战鹰的[入门教程](https://www.bilibili.com/video/BV16g411q7qA/?spm_id_from=333.999.0.0)），就像家长学校在孩子成长初期总结的各种定式一样。他们都是一种稳健的方法，并没有仔细地考察具体的情况，比如小时候就是给所有孩子的建议都会有好好学习数学，但是每个人的初始条件毕竟还是差异较大。机器有更强大的算力，可以充分考虑此时此刻的全貌，并且可以计算到这些局部最优行为的长期影响，这样机器就可以给出非常个性化的策略，这是人力难及的。

**Reply by yufree** [reply]
只能说有可能，从算法上只要不是穷举是不是最优很难判断，Alphago算法也不是穷举，更多是高效寻优。围棋属于局面上子越下越多的博弈，越接近官子越容易算最优，但布局阶段棋盘几乎空白就不好算了，后面变数太大，机器也不会算清全貌，但对局训练多了也会对某些定式赋予高胜率，这点应该跟人差不多，只是机器的训练量远超人类，初始扰动足够的情况下是能发现新定式的。所以我觉得机器的确比人强，但可能存在比机器更强的智慧，从开局就能把对手跟自己对策全算清，稳赢。类似解析解与数值解，机器跟人基于经验或训练算的都属于是数值解，但应该有个终极解析解，计算复杂度上恐怕都不是图灵机构架能算的。
